### PR TITLE
Makefile: suffix test log files

### PR DESCRIPTION
**Pull request by gyuho**
In preparation of running all tests inside container.
Currently, we run Jenkins in shared environment.
This is not good. Need manual Go runtime updates,
cannot run two different branches, port conflicts,
out of disk on /tmp, etc.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,20 +1,27 @@
-# run makefile from repo root
+# run from repository root
+
+TEST_SUFFIX = $(shell date +%s | base64 | head -c 15)
```
Comment (xiang90) [inline Makefile]: probably needs to echo out the test_suffix. or it would not be easy to figure out which log belongs to which test when something fails.

**Comment by xiang90** [comment]
do you know why jenkins-cov and jenkins-ppc64le keep on failing?

lgtm.

**Comment by gyuho** [comment]
@xiang90 Yeah, been failing for a few days. Will email the maintainer. coverage tests are failing from `out of disk` errors... Need to move these to container.

**Comment by xiang90** [comment]
>  coverage tests are failing from out of disk errors...

can we clean up logs and probably auto remove very old logs when finishing up a test?

**Comment by gyuho** [comment]
We already run this

```
# delete tmp files older than a day
find /tmp/ -mtime +2 -maxdepth 1 | grep -v "/tmp/$" | xargs rm -rf
```

But sometimes one day is not enough.

**Comment by xiang90** [comment]
@gyuho that is strange. can you check what is the size of the log we generate yesterday and what is the disk capacity?

**Comment by gyuho** [comment]
@xiang90 You are right. Maybe just temporarily out of disk during compilation.

```
# testmain
/usr/local/go/pkg/tool/linux_amd64/link: flushing /tmp/go-link-341148514/go.o: write /tmp/go-link-341148514/go.o: no space left on device
```

I will investigate more.

File: Makefile (hunk 1 above)
Comment (gyuho) [reply Makefile]: Added.

**Pull request merged**